src/mgmt/operator/workflows/droplets/triggers.py

```diff
@@ -58,7 +58,7 @@ def droplet_opr_on_droplet_provisioned(body, spec, **kwargs):
 @kopf.on.delete(group, version, RESOURCES.droplets, retries=OBJ_DEFAULTS.kopf_max_retries)
 def droplet_opr_on_droplet_delete(body, spec, **kwargs):
     param = HandlerParam()
-    param.name = kwargs['name']
+    param.name = spec['name']
```

Comment:
Not sure why this needs to be changed, but if we change it here can we change it everywhere else to be consistent?

Reply:
fixed